src/app/components/home/home.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="header-nav">
-  <div routerLink="/splash"><img class="logo" src="../../../assets/home/VV_Logo_Reversed@2x.png" alt="Voter's Voice Initiative" width="120"
+  <div routerLinkActive="active-link" [routerLink]="[{ outlets: {sidenav: ['splash']} }]"><img class="logo" src="../../../assets/home/VV_Logo_Reversed@2x.png" alt="Voter's Voice Initiative" width="120"
```

Comment:
With the aforementioned changes for the app routing module, the initial `routerLink="/spash"` should work.  We no longer need to do the whole outlets: {sidenav: ...} format.  All you need to do is to replace the routerLinks, without the brackets, with their corresponding routes.

And for the <router-outlet>, you can also remove the `name="sidenav"`.

Reply:
@gSoD17 Fixed this as well. Check it out.